### DISCUSSION TITLE
Can't Login

### DISCUSSION THREAD

**Discussion by jsfowles** [vercel/community#2829] [Help]
### Summary

I've had my Vercel account for years and now I'm getting the following error when attempting to login through Github
<img width="408" alt="CleanShot 2023-06-12 at 11 41 38@2x" src="https://github.com/vercel/community/assets/17182194/a9aae790-59fc-4aba-8a15-1fe4443e5234">

### Steps to Reproduce

Click login, Click Connect with Github, see error.<img width="408" alt="CleanShot 2023-06-12 at 11 41 38@2x" src="https://github.com/vercel/community/assets/17182194/a9aae790-59fc-4aba-8a15-1fe4443e5234">

**Comment by amyegan** [comment]
Hi @jsfowles. Is it linked to the GitHub account you're using for this discussion, or is there a different username that I should use to check the account?

**Reply by jsfowles** [reply]
Yes, it's linked to this account. I've been working with someone via registration@vercel.com but it hasn't been very helpful.

**Reply by zapbra** [reply]
I have the same issue. Try to create a new account with the same email as ur main account and it logs you in or sign in through your email.

**Comment by jsfowles** [comment]
It looks like I have two accounts on where I can login with my email and one I can login using Github, both use the same email address. @amyegan could this be the issue? I really need this to be resolved, sorry to bug!

**Reply by amyegan** [reply]
Sorry this still isn't solved for you, Jacob. Two accounts with the same email address does seem like it would cause some confusion. If you let me know the case number I can look into it 🙇

**Reply by jsfowles** [reply]
Case # is 00141041. The last response I received was basically, sorry nothing we can do. Which would be very disappointing and a massive amount of work for me to redeploy all my projects.

**Reply by jsfowles** [reply]
There was an issue with my email address and a rep from Vercel took care of it. Thank you for your willingness to help.

**Comment by jsfowles** [comment]
Vercel representative fixed the issue.
